Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "Synth",
```

Comment:
can you rename all instances of library name from `Synth` to `synth-ios`

Reply:
Renamed all instances to synth-ios in commit c7fc339